atom/common/application_info.cc

```diff
@@ -4,7 +4,29 @@
 
 #include "atom/common/application_info.h"
 
+#include "atom/browser/browser.h"
+#include "atom/common/atom_version.h"
 #include "base/no_destructor.h"
+#include "base/strings/stringprintf.h"
+#include "chrome/common/chrome_version.h"
+#include "content/public/common/user_agent.h"
+
+std::string GetUserAgent() {
```

Comment:
this should perhaps go in the `atom` namespace?

Reply:
Fixed.